data/saas/dataset/stripe_dataset.yml

```diff
@@ -0,0 +1,1505 @@
+              data_categories: [system.operations]
+            - name: uncategorized_text
+              data_categories: [system.operations]
+          - name: evidence_details
+            fields:
+            - name: due_by
+              data_categories: [system.operations]
+            - name: has_evidence
+              data_categories: [system.operations]
+            - name: past_due
+              data_categories: [system.operations]
+            - name: submission_count
+              data_categories: [system.operations]
+          - name: is_charge_refundable
+            data_categories: [system.operations]
+          - name: livemode
+            data_categories: [system.operations]
+          - name: payment_intent
+            data_categories: [system.operations]
+          - name: reason
+            data_categories: [system.operations]
+          - name: status
+            data_categories: [system.operations]
+      - name: payment_intent
+        fields:
+        - name: id
+          data_categories: [user.derived.identifiable.unique_id]
+          fidesops_meta:
+            primary_key: True
+            data_type: string
+        - name: object
+          data_categories: [system.operations]
+        - name: amount
+          data_categories: [user.provided.identifiable.financial]
+          fidesops_meta:
+            data_type: integer
+        - name: amount_capturable
+          data_categories: [user.provided.identifiable.financial]
+          fidesops_meta:
+            data_type: integer
+        - name: amount_received
+          data_categories: [user.provided.identifiable.financial]
+          fidesops_meta:
+            data_type: integer
+        - name: application
+          data_categories: [system.operations]
+        - name: application_fee_amount
+          data_categories: [system.operations]
+        - name: automatic_payment_methods
+          data_categories: [system.operations]
+        - name: canceled_at
+          data_categories: [system.operations]
+        - name: cancellation_reason
+          data_categories: [system.operations]
+        - name: capture_method
+          data_categories: [system.operations]
+        - name: client_secret
+          data_categories: [system.operations]
+        - name: confirmation_method
+          data_categories: [system.operations]
+        - name: created
+          data_categories: [system.operations]
+        - name: currency
+          data_categories: [system.operations]
+        - name: customer
+          data_categories: [system.operations]
+        - name: description
+          data_categories: [system.operations]
+        - name: invoice
+          data_categories: [system.operations]
+        - name: last_payment_error
+          data_categories: [system.operations]
+        - name: livemode
+          data_categories: [system.operations]
+        - name: next_action
+          data_categories: [system.operations]
```

Comment:
After annotating most of these fields with system.operations, I think it would be more efficient for us to just annotate the PII fields and expand our annotations as needed. We can include this along with your previous comment about `user.derived.identifiable.unique_id` in a SaaS connector dev guide after reviewing with the team.